compiler-rt/lib/sanitizer_common/sanitizer_common_interceptors.inc

```diff
@@ -6304,10 +6304,38 @@ INTERCEPTOR(int, fclose, __sanitizer_FILE *fp) {
 #endif
 
 #if SANITIZER_INTERCEPT_DLOPEN_DLCLOSE
+// Returns 1 if key is a suffix of str, 0 otherwise
+static int internal_strcmp_suffix(const char *key, const char *str) {
```

Comment:
One confusing part for this implementation is that it has the opposite return value convention of strcmp. Maybe rename to internal_is_suffix? Also key can be renamed to suffix.

Reply:
Good idea, thanks! Done in latest patch set.